File: C/pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int i,h,j;
    do
    {
        h=get_int("Height:");
    }while(h<1||h>8);
    for(i=0;i<h;i++)
    {
        for(j=h-1-i;j>0;j--)
        {
            printf(" ");
        }
        for(j=0;j<i+1;j++)
        {
            printf("#");
        }
        printf("  ");
        for(j=0;j<i+1;j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Python/pset6/mario.py
from cs50 import get_int

h = 10
while h <= 0 or h > 8:
    h = get_int("Height: ")

for i in range(h):
    for j in range(h - i - 1):
        print(" ", end = "")
    for j in range(i + 1):
        print("#", end = "")
    print("  ", end = "")
    for j in range(i + 1):
        print("#", end = "")
    print("")